<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbManagerQueryBuilderDlg</class>
 <widget class="QDialog" name="DbManagerQueryBuilderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL query builder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Columns</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="col">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tables     </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tab">
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="where"/>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="group">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="order">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Where    </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Group by</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Order by</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>250</width>
            <height>313</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Data</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Show system tables</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="tables">
             <item>
              <property name="text">
               <string>Tables</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="columns">
             <item>
              <property name="text">
               <string>Columns</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="aggregates">
             <item>
              <property name="text">
               <string>Aggregates</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="functions">
             <item>
              <property name="text">
               <string>Functions</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="math">
             <item>
              <property name="text">
               <string>Math</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="stringfct">
             <item>
              <property name="text">
               <string>Strings functions</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="operators">
             <item>
              <property name="text">
               <string>Operators</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>250</width>
            <height>313</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Columns' values</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QComboBox" name="columns_2">
             <item>
              <property name="text">
               <string>Columns</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extract">
             <property name="text">
              <string>Only 10 first values</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="values">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>250</width>
            <height>313</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Spatial index</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QComboBox" name="table_idx">
             <item>
              <property name="text">
               <string>Table (with spatial index)</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="table_target">
             <item>
              <property name="text">
               <string>Table (Target)</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="usertree">
             <property name="text">
              <string>Use spatial index</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbManagerQueryBuilderDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbManagerQueryBuilderDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>where</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>tab</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>col</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>columns</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
